src/main/java/Number.java

```diff
@@ -0,0 +1,24 @@
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Number {
+    private static final int NUMBER_LENGTH_LIMIT = 3;
+    public static final int MAX_VALUE_OF_BASEBALL_NUMBER = 9;
+    public static final int MIN_VALUE_OF_BASEBALL_NUMBER = 1;
+
+    public String create() {
+        List<Integer> numbers = new ArrayList<>();
+        int currentNumberLength = 0;
+        while (currentNumberLength < NUMBER_LENGTH_LIMIT) {
+            int number = (int) (MIN_VALUE_OF_BASEBALL_NUMBER + Math.random() * MAX_VALUE_OF_BASEBALL_NUMBER);
+            if (!numbers.contains(number)) {
+                numbers.add(number);
+                currentNumberLength = currentNumberLength + 1;
```

Comment:
```suggestion
        while (numbers.size() < NUMBER_LENGTH_LIMIT) {
            int number = (int) (MIN_VALUE_OF_BASEBALL_NUMBER + Math.random() * MAX_VALUE_OF_BASEBALL_NUMBER);
            if (!numbers.contains(number)) {
                numbers.add(number);
```
이 부분은 이렇게 변경하는게 더 좋지 않을까요? 🤔

Reply:
불필요한 변수 할당을 없앨 수 있는 것 같습니다. 좋은 지적 감사합니다.